sample/Getting started.jl

```diff
@@ -13,23 +13,29 @@ macro bind(def, element)
     end
 end
 
+# ╔═╡ 9b62f2c6-8037-486f-b8ec-a95b577b8d74
+# Code targeted for potential inclusion in the Pluto.jl package:
+begin
+	using Pkg
+	Pkg.add(PackageSpec(url="https://github.com/MichaelHatherly/CommonMark.jl",
+	rev="mh/interp"))
```

Comment:
Should be available on latest release btw @jeremiahpslewis.

Reply:
Fixed, think this is no longer necessary thanks to the package manager!